crates/rethnet_evm_napi/Cargo.toml

```diff
@@ -9,7 +9,7 @@ crate-type = ["cdylib"]
 [dependencies]
 anyhow = "1.0.64"
 crossbeam-channel = { version = "0.5.6", default-features = false }
-napi = { version = "2.10.1", default-features = false, features = ["async", "error_anyhow", "napi8", "serde-json"] }
+napi = { version = "= 2.10.2", default-features = false, features = ["async", "error_anyhow", "napi8", "serde-json"] }
```

Comment:
I had to pin the `napi` version as a bug was introduced in `2.10.3` that breaks our build:
https://github.com/napi-rs/napi-rs/issues/1401